src/components/graphQLConnect.ts

```diff
@@ -28,6 +28,27 @@ class GraphQLdb {
     return data.data
   }
 
+  public getReviewData = async (reviewId: number) => {
+    const data = await this.gqlQuery(
+      `
+      query getReviewData($id: Int!) {
+        review(id: $id) {
+          level
+          isLastLevel
+          status
```

Comment:
Hmm, the only problem with using a GraphQL query here rather than SQL is that the status comes through as "CHANGES_REQUESTED", rather than "Changes Requested". I guess it's fine, we just need to be careful with this kind of thing -- they're not completely interchangeable and might run into a problem when updating Review status later on.

Reply:
yeah. nicely spotted, as per convo, lets replace postgresql ones with capital and underscore

Reply:
#241